repo.go

```diff
@@ -167,6 +167,40 @@ func (r *Repo) RootVersion() (int, error) {
 	return root.Version, nil
 }
 
+func (r *Repo) GetThreshold(keyRole string) (int, error) {
```

Comment:
Is this function only used for testing? I have no real objection to that, but in the future it might be nice to be able to check the current thresholds from the cli.

Reply:
Oh yeah, I'll expose a `get-threshold` in the CLI. I figured it'd be a useful getter in using the Repo implementation